shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -236,6 +236,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
 
         result.setSelected(object.hasFlag(Flag.IS_SELECTED));
         result.setText(object.getValueLabelHint());
+        result.setTraversalBefore(mOwner, object.nextNodeId);
```

Comment:
Should we do this only if `nextNodeId != -1` (or if that is guaranteed to be true, should there be an assert somewhere for that?)

Reply:
It so happens that View.NO_ID is what you put when there isn't a next node, and that is also -1. I added an inline conditional, however, to check for -1 and convert it to View.NO_ID, in case that value ever changes.